src/EnergyPlus/api/datatransfer.cc

```diff
@@ -829,7 +829,7 @@ int getConstructionHandle(EnergyPlusState state, const char *constructionName)
             return handle;
         }
     }
-    return 0;
+    return -1; // return -1 if it wasn't found
```

Comment:
getConstructionHandle was returning 0 instead of -1, unlike other similar methods + the python user facing docs

https://github.com/NREL/EnergyPlus/issues/9998#issuecomment-1534398312

Reply:
OK